SQL/0000-00-04-Help.sql

```diff
@@ -70,4 +70,5 @@ INSERT INTO `help` (helpID, parentID, hash, topic, content, created, updated) VA
 (48,-1,md5('genomic_browser'), 'Genomic Browser', 'The Genomic Browser houses analyzed genetic results and facilitates exploration of these datasets, cross-linked with behavioural and imaging subject data.  Within each tab, all fields or only a selection of Summary fields can be viewed by toggling the Display select box.  One-click download of the current dataview is also available.\r\nThe Profiles tab displays all genetic data and files stored in Loris per candidate, and enables application of filters.\r\nThe GWAS, SNP, CNV and Methylation tabs enable filtering and exploration of analysis-specific datasets performed on a candidate population.\r\nThe Files tab displays all genetic filesets available for download, and provides upload functionality for Methylation data (LORIS 16.0). Clicking on a fileset name will load a subpage displaying the metadata available for this file, as well enabling its download.\r\n', '2016-04-07 00:00:00', NULL),
 (49,38,md5('project'), 'Project', 'You then click on &quot;New ProjectID&quot;, fill in the fields on the right, click save.  Immediately refresh the page to view your new project.  Clicking save more than once will register a duplicate project ID.  If you create an extra ID, you have to delete it from the database manually with an sql command.\n\nDefine all projectID-subprojectID relationships by populating the project_rel table, e.g.\n\nINSERT INTO `project_rel` VALUES (1,1),(1,2),(2,3);', '2016-04-01 00:00:00', NULL),
-(50,38,md5('subproject'), 'Subproject', 'You then click on &quot;New SubprojectID&quot;, fill in the fields on the right, click save.  Immediately refresh the page to view your new subproject.  Clicking save more than once will register a duplicate subproject ID.  If you create an extra ID, you have to delete it from the database manually with an sql command.\n\nDefine all projectID-subprojectID relationships by populating the project_rel table, e.g.\n\nINSERT INTO `project_rel` VALUES (1,1),(1,2),(2,3);', '2016-04-01 00:00:00', NULL);
+(50,38,md5('subproject'), 'Subproject', 'You then click on &quot;New SubprojectID&quot;, fill in the fields on the right, click save.  Immediately refresh the page to view your new subproject.  Clicking save more than once will register a duplicate subproject ID.  If you create an extra ID, you have to delete it from the database manually with an sql command.\n\nDefine all projectID-subprojectID relationships by populating the project_rel table, e.g.\n\nINSERT INTO `project_rel` VALUES (1,1),(1,2),(2,3);', '2016-04-01 00:00:00', NULL),
```

Comment:
Will the HTML entity tags get replaced with &amp;quot; upon save in the frontend since the database wrapper tries to sanitize HTML in user input?

(Either way, they're already there, so this won't make it any worse than it is..)